### PR TITLE
Manfred circos add panzoom

### PR DESCRIPTION
**Pull request by mj3cheun**
## About
- [ ] This is a new component
- [X] I am adding a feature to an existing component, or improving an existing feature
- [ ] I am closing an issue

## Description of changes
Adds panzoom. Note: It is not possible to change panzoom on/off and svg download on/off once component is initialized. This is a limitation of the panzoom and svg download implementation on the circosJS side. This can be addressed in another update.

## Before merging
- [ ] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [ ] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [ ] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by shammamah** [comment]
@mj3cheun Please build the package, publish an rc version to PyPI, and put that in the `requirements.txt` (so that the demo app will reflect the changes you've made). Also remember to put in a CHANGELOG entry!

**Comment by mj3cheun** [comment]
Let me know when its good and i will merge and rebuild :)

File: dash_bio/metadata.json [hunk 1]
```diff
@@ -980,7 +1004,7 @@
         "returns": null
       },
       {
-        "name": "configIdeogram",
+        "name": "configCircos",
```
Comment (shammamah) [inline dash_bio/metadata.json]: Why is this change here?

File: src/lib/components/Circos.react.js [hunk 2]
```diff
@@ -15,10 +15,25 @@ export default class Circos extends Component {
     constructor(props) {
         super(props);
         this.circos = null;
-        this.configIdeogram = this.configIdeogram.bind(this);
```
Comment (shammamah) [inline src/lib/components/Circos.react.js]: Oh, I see. Super weird that this was in here to begin with.

File: src/lib/components/Circos.react.js [hunk 3]
```diff
@@ -391,30 +402,30 @@ Circos.propTypes = {
...
-	 * Ex: This will return (source)(sourceID) + ' > ' + (target)(targetID) + ': ' (target)(targetEnd)'.
-	 * "tooltipContent": {
-	    "source": "source",
-	    "sourceID": "id",
-	    "target": "target",
-	    "targetID": "id",
-	    "targetEnd": "end"
-	}
-	 **/
+             * Specify what data for tooltipContent is
```
Comment (shammamah) [inline src/lib/components/Circos.react.js]: Do these whitespace changes come from your linter?

**Comment by shammamah** [comment]
Looks great! Please approve https://github.com/plotly/dash-bio/pull/384 so we can get it merged in, then add a CHANGELOG entry/bump the rc version/republish to PyPI. I think after this one gets merged in, we will be good to release `0.1.2`.

File: dash_bio/metadata.json (hunk 1 above)
Comment (mj3cheun) [reply dash_bio/metadata.json]: this change came from the build

File: src/lib/components/Circos.react.js (hunk 2 above)
Comment (mj3cheun) [reply src/lib/components/Circos.react.js]: lol i just assumed that that was meant to be `circos` not `ideogram`

File: src/lib/components/Circos.react.js (hunk 3 above)
Comment (mj3cheun) [reply src/lib/components/Circos.react.js]: No i made these changes. The whitespace looked to be messed up so i corrected it.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,9 +7,11 @@
 * Optimized Speck component further to increase performance.
 * Updated dash_bio_utils to version that introduced standardization of
   data parsers.
+* Changed circos source from npm to github fork.
```
Comment (shammamah) [inline CHANGELOG.md]:
```suggestion
* Changed Circos source from NPM to GitHub fork.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,9 +7,11 @@
...
 
 ### Added
 * Added default props to Molecule3dViewer component.
+* Added zoompan and export SVG to circos.
```
Comment (shammamah) [inline CHANGELOG.md]:
```suggestion
* Added zoom/pan and export SVG capabilities to Circos.
```
